spaceship.zsh

```diff
@@ -244,6 +256,62 @@ spaceship_ruby_version() {
   echo -n "%{$reset_color%}"
 }
 
+# Swift
+# Show current version of Swift
+spaceship_swift_version() {
+
+  if [[ $SPACESHIP_SWIFT_SHOW_GLOBAL == true ]] ; then
+    if command -v swiftenv > /dev/null 2>&1; then
```

Comment:
You're checking twice if `swiftenv` exists. You definitely know about DRY principle, right? I would suggest you to replace these conditionals by single line on top of `spaceship_swift_version()` function:

```zsh
command -v swiftenv > /dev/null 2>&1 || return
```

This line means: if there is no `swiftenv`, then there is no reason to execute whole function, so we just `return` out of there.